pkg/scheduler/framework/plugins/interpodaffinity/filtering_test.go

```diff
@@ -268,7 +270,7 @@ func TestRequiredAffinitySingleNode(t *testing.T) {
 					PodAntiAffinityIn("service", "zone", []string{"securityscan", "value2"}, st.PodAntiAffinityWithRequiredReq).Obj(),
 			},
 			node: &node1,
-			wantStatus: framework.NewStatus(
```

Comment:
This test case confirms even if no Affinity on incoming Pod, PreFilter won't return Skip if any existing Pod has AntiAffinity which match with incoming Pod.